### ISSUE TITLE
paper-tab: disabled option partially work

### ISSUE THREAD

**Issue by panthony** [miguelcobain/ember-paper#791]
When you set `disabled=true` on a tab. It has a "disabled" look & feel but you are still able to click on it and it gets the selected state.

The `Basic Usage` should have one of its tab disabled to have an exemple of this option visible.

**Comment by BennyAlex** [comment]
@miguelcobain @panthony 
When will this fixed? It's a really annyoing error

**Comment by panthony** [comment]
@BennyAlex I may take a look at this eventually if nobody does but for now I'm happy with the following workaround:

```hbs
  {{#tabs.tab value=route disabled=disabled href=(if (not disabled) (href-to route))}}
     Label
  {{/tabs.tab}}
```

**Comment by BennyAlex** [comment]
@panthony This will only work when using route-tabs, not when just using the index

**Comment by BennyAlex** [comment]
@miguelcobain 
hey, are you able to release a bugfix?
The tabs working fine exept this bug :/

**Comment by miguelcobain** [comment]
Closed via #971
